CHANGELOG.md

```diff
@@ -1,4 +1,59 @@
-## 1.9.2 (Upcoming)
+## 1.9.2 (July 19, 2023)
+
+### NOTES:
+
+* Vendored plugins within Packer have not been updated. Plugin releases occur on
+     a regular basis to address issues and feature requests.
+     Please note that in an upcoming version of Packer, we will remove the last
+     bundled plugins from Packer.
+     Users are encouraged to use `packer init` for HCL2 templates or
+     `packer plugins install` with legacy JSON templates for installing external
+     plugins.
+
+* Packer will now warn when using bundled plugins. This feature will be removed in
+    a future version of the tool, so this warning is meant to bring awareness of the
+    upcoming change, and to help users update their environment/templates to be
+    ready for this.
+    [GH-12495](https://github.com/hashicorp/packer/pull/12495)
+
+### BUG FIXES:
+
+* Fixed a bug with how Packer was discovering plugins: in order to load plugins, Packer
+    would recursively scan all the known directories in which we could have installed
+    plugins. This caused a lot of unnecessary directory walks, which if the storage
+    was slow (HDD, NAS, etc.) caused major slowdowns upon invocation.
+    This has been reworked so Packer only scans directories in which plugins are
+    installed through commands such as `packer init` or `packer plugins install`,
+    which considerably reduces the number of directories scanned.
+    [GH-12414](https://github.com/hashicorp/packer/pull/12414)
```

Comment:
Slight reword to remove call out of slow storage and clarify the messaging around nested plugin scanning. I don't want to give the impression that Packer will no longer check the executable or current working directories for a single plugin. 
```suggestion
 * Fixed a bug with how Packer was discovering plugins: in order to load
      plugins, Packer would recursively scan all the known directories in which
      we could have installed plugins. This caused unnecessary directory
      walks and slowdowns upon invocation. Packer will now only check
      for nested plugins within the directories used by commands such as `packer
      init`, or `packer plugins install`, or as defined in PACKER_PLUGIN_PATH.
      Refer to the [Packer's plugin directory documentation](https://developer.hashcor.com/packer/docs/configure#packer-s-plugin-directory) for details on how loading works.
      [GH-12414](https://github.com/hashicorp/packer/pull/12414)
```

Reply:
Looks good to me, I rerolled the PR with this changed.